tools/fuchsia/devshell/serve.sh

```diff
@@ -174,9 +174,10 @@ while true; do
     fi
 
     config_url="http://${addr}:${port}/config.json"
-    run_ssh_command amberctl add_src \
+    run_ssh_command pkgctl add \
       -n "engine" \
-      -f "${config_url}"
+      # We need backwards compatibility with V1 configs for V1 runners.
```

Comment:
The v1 config is not related to the version of the component but rather the version of the config.json file that is created by the host side package server. Since we control the package server and the configuration of that server on the device we can move to the v2 version.

This would mean that we would have to pass the `-c 2` flag to the call to `pm serve` on line 63. When you do that you can remove the `-f 1` flag from the add call.

Reply:
I see, I think I misunderstood a comment on https://github.com/flutter/flutter/pull/89785. Thanks, fixed.